src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -704,9 +698,8 @@ bool clang_c_convertert::get_function_param(
   param.type() = param_type;
   param.cmt_base_name(name);
 
-  if(name.empty())
-    if(!name_param_and_continue(pd, id, name, param))
-      return false;
+  if(name.empty() && name_param_and_continue(pd, id, name, param))
```

Comment:
It would be good to know what the return value of this `name_param_and_continue()` function means: What are "failure" or "success" conditions? Unfortunately, the comment inside its definition isn't clear about that at all and all possible paths through that function (and its overrides) currently return `false`.

Reply:
I agree, maybe `get_param_name` to keep it in line with the other function names?

Reply:
True, we probably should return false if `get_cpyctor_name` manages to populates the id and name, otherwise return true to indicate something goes wrong. Also `get_cpyctor_name` probably needs some return values to indicate good or bad, just like other functions.

Reply:
Hi, Thank you very much for your suggestions. 

I want to express my thoughts, this method did return false by default before, When I looked at the structure of the code again I thought it might make sense for us to change this method to `viod`, since there is no failure case in this method (we always get the name, and we don't need the name for non-constructors and operators).

This is the first time I've adjusted the front-end code, so please point me out if anything is wrong.

Reply:
I'm ok with void if this function doesn't really need to return a status. I personally didn't find "ERROR: CONVERSION ERROR" helpful for debugging.